bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/thq_4f_ckm_LO_ctcvcp/thq_4f_ckm_LO_ctcvcp_run_card.dat

```diff
@@ -56,7 +70,7 @@
   1	=  alpsfact          ! scale factor for QCD emission vx
   False	=  chcluster         ! cluster only according to channel diag
   True	=  pdfwgt            ! for ickkw=1, perform pdf reweighting
-  4	=  asrwgtflavor      ! highest quark flavor for a_s reweight
+  5	=  asrwgtflavor      ! highest quark flavor for a_s reweight
```

Comment:
although this block will not be called due to no matching, it should be 4 in 4 flavor scheme. btw, line 63, "0.33 =  scalefact" is set deliberately, right?

Reply:
Hi, 

thanks for your check. I updated it to `4	=  asrwgtflavor` and set the `scalefact` to 1. It was by mistake used from the previous version.